packages/contracts/contracts/universal/Semver.sol

```diff
@@ -43,7 +43,7 @@ contract Semver {
      *
      * @return Semver contract version as a string.
      */
-    function version() public view returns (string memory) {
+    function version() public virtual view returns (string memory) {
```

Comment:
The overriding is necessary because the GovernorUpgradeable contract has the same version function.
This is a choice to use the functionality of semver without modifying the GovernorUpgradeable contract.